5-sprint-mission/src/services/productService.ts

```diff
@@ -60,7 +61,17 @@ export async function getById(id: number) {
 }
 
 export async function update(productData: ProductUpdateData) {
-  return updateData(productData);
+  const existing = await getByIdData(productData.id);
+  const updated = await updateData(productData);
+
+  if (
+    existing?.price !== undefined &&
+    productData.price !== undefined &&
+    existing.price !== productData.price
```

Comment:
```suggestion
    existing?.price &&
    productData.price &&
    existing.price !== productData.price
```
undefined가 falsy 니까 이렇게 수정 할 수도 있을 것 같아요